src/modules/transaction/types.ts

```diff
@@ -78,22 +88,33 @@ export type Transaction = {
   from: string
   actionType: string
   status: AnyTransaction['status'] | null
+  url: string
+  isCrossChain: boolean
+  requestId?: string
+  toChainId?: ChainId
+  crossChainProviderType?: CrossChainProviderType
   withReceipt?: boolean
   receipt?: { logs: TransactionReceipt['logs'] }
   payload?: any
   chainId: ChainId
 }
 
 export type TransactionPayload = {
-  [hash: string]: {
+  [key: string]: any,
+  [TRANSACTION_ACTION_FLAG]: {
     chainId: ChainId
+    toChainId: ChainId
```

Comment:
just a question, will this be a breaking change? this field is now required (just a comment so you know when releasing)

Reply:
Made it optional because there surely are lots of transactions stored in the local storages without it.